qucs/qucs/dialogs/sweepdialog.cpp

```diff
@@ -256,8 +238,34 @@ Graph* SweepDialog::setBiasPoints()
         }
     }
 
+    if(pc->Model == "WProbe") {
+      pn = pc->Ports.at(0)->Connection;
+      if(!pn->Name.isEmpty())   // preserve node voltage ?
+      //  pn = pc->Ports.next()->Connection;
+	pn = pc->Ports.at(1)->Connection;
+      pn->x1 = 0x10;   // mark current
+      pg->Var = pc->Name + ".I";
+      if(Diag->loadVarData(DataSet, pg)) {
+        pn->Name = num2str(*(pg->cPointsY)) + "A";
+        NodeList.append(pn);             // remember node ...
+        ValueList.append(pg->cPointsY);  // ... and all of its values
+        pg->cPointsY = 0;   // do not delete it next time !
+      }
+      else
+        pn->Name = "0A";
```

Comment:
Again, this block is seems to be the only part related to your watt-meter probe.
All the other changes are revering lines on latest master. We cannot have that.